lib/js/_utils/_check-current-version.js

```diff
@@ -15,37 +15,33 @@ const getLatestVersion = require('./_get-latest-version');
  * Compare package.json version and NPM version
  * @async
  * @function checkCurrentVersion
- * @param {Object} pathToPackage package path on filesystem
+ * @param {String} pathToPackage package path on filesystem
  * @param {Boolean} printTitle log the title to the CLI
  * @return {Promise<Boolean>}
  */
 async function checkCurrentVersion(pathToPackage, printTitle) {
-	const pkg = require(`${pathToPackage}/package.json`);
-	const newVersion = checkNewVersion(pkg);
+	const package_ = require(`${pathToPackage}/package.json`);
```

Comment:
why the underscore at the end of the const name?

Reply:
`package` is reserved in node and this got picked up by the new linting rules that came with updating the deps